spec/system/tech_shop_spec.rb

```diff
@@ -37,150 +39,142 @@
     # Subproducts
     @camera = Camera.create!(name: 'canon eos rebel t3', href: 'canon')
 
-    100.times { Product.create!(name: 'crapoola', href: 'crappy', tags: ['crappy']) }
-
-    @products_in_database = Product.all
```

Comment:
This should probably save on memory, but the biggest difference it'll probably make is removing a line of code that we never needed.